hack/make/go.mk

```diff
@@ -1,39 +1,13 @@
-LINT_TARGET ?= ./...
-
-ifeq ($(LINT_TARGET), ./...)
-	GCI_TARGET ?= .
-else
-	GCI_TARGET ?= $(LINT_TARGET)
-endif
-
-## Runs go fmt
-go/fmt:
-	go fmt $(LINT_TARGET)
-
-## Runs gci
-go/gci:
-	gci write --skip-generated $(GCI_TARGET)
-
-## Runs linters that format/change the code in place
-go/format: go/fmt go/gci
-
-## Runs go vet
-go/vet:
-	go vet -copylocks=false $(LINT_TARGET)
-
-## Runs go wsl linter
-go/wsl:
-	wsl -fix -allow-trailing-comment ./pkg/...
```

Comment:
question: these linters/lines are here because they not only validate but also fix problems, so that's why we have a flag `--write` by removing them we need to do it manually or we need to introduce: `golangci-lint run --fix`
where  **--fix** - Fix found issues (if it's supported by the linter)`